packages/tools/pie-icons/package.json

```diff
@@ -18,9 +18,9 @@
     "prepare": "yarn build && yarn test:coverage",
     "clean": "rimraf -rf ./dist/*",
     "prepublishOnly": "yarn run build",
-    "lint:scripts": "eslint .",
-    "lint:scripts:fix": "eslint . --fix",
-    "test": "jest",
+    "lint:scripts": "run -T eslint .",
```

Comment:
What does `run -T` do?

Reply:
@fernandofranca It's a little-known flag that we only recently discovered! https://yarnpkg.com/cli/run#options

In this case it looks for the eslint dependency in the root of the monorepo.

Reply:
That's awesome! 🔥